HLTrigger/Tools/test/testConvertToRaw.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Pass in name and status
+function die {
+  printf "\n%s: status %s\n" "$1" "$2"
+  exit $2
+}
+
+check_for_success() {
+    "${@}" && echo -e "\n ---> Passed test of '${@}'\n\n" || exit 1
+}
+
+check_for_failure() {
+    "${@}" && exit 1 || echo -e "\n ---> Passed test of '${@}'\n\n"
+}
+
+echo "========================================"
+echo "Testing convertToRaw in ${SCRAM_TEST_PATH}."
+echo "----------------------------------------"
+echo
+
+echo "========================================"
+echo "testing help function "
+echo "----------------------------------------"
+echo
+
+convertToRaw --help  || die "Failure running convertToRaw --help" $?
+
+echo "========================================"
+echo "testing successful conversion"
+echo "----------------------------------------"
+echo
+
+check_for_success convertToRaw -f 1 -l=1 -v root://eoscms.cern.ch//eos/cms/store/data/Run2024C/EphemeralHLTPhysics0/RAW/v1/000/379/416/00000/e8dd5e3c-216f-4545-acb6-ab86c9161085.root
```

Comment:
I am not sure if this is OK, but without the re-director `root://eoscms.cern.ch//eos/cms/` the test was failing: [log](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-44a4e8/38926/unitTests/src/HLTrigger/Tools/test/testConvertToRaw/testing.log)

I would presume that once the file is cached we can remove the re-director. I don't know if there's a way of caching the file upfront (tagging @smuzaffar for suggestions - the file size is 900MB, hope we're ok with it).

Reply:
@mmusich I think the size is OK (Shahzad can correct me if needed),and regarding caching this file - see similar discussion here: https://github.com/cms-sw/cmssw/issues/44293#issuecomment-1976849048 .

Reply:
@mmusich  , this data file is now cached. Please use logical file name e.g. `convertToRaw -f 1 -l=1 -v /store/data/Run2024C/EphemeralHLTPhysics0/RAW/v1/000/379/416/00000/e8dd5e3c-216f-4545-acb6-ab86c9161085.root` . This should work for PR/IB tests. You can locally test it by setting ibeos cache paths
```
export CMS_PATH=/cvmfs/cms-ib.cern.ch
export SITECONFIG_PATH=/cvmfs/cms-ib.cern.ch/SITECONF/local
```

Reply:
thank you. Done in the last push.